raiden/raiden_event_handler.py

```diff
@@ -67,6 +68,12 @@
 )
 
 
+def storage(raiden) -> SQLiteStorage:
```

Comment:
What problem does this solve?

Reply:
I guess adds the unrecoverable error and makes it non-optional since mypy must have been complaining it's Optional?

Reply:
But I also don't like it. It introduces a dependency of the `raiden_event_handler.py` module to the `storage.sqlite` module and I think a simple assert at the beginning of the function can solve this better.

It should not be optional anyway ... it's only optional if accesed before raiden has started. Perhaps we can type it better too. I will have a look.

Reply:
Done here: https://github.com/raiden-network/raiden/pull/3916/commits/4054ed2eef49ba880c04795934802dad3b82d83c and here: https://github.com/raiden-network/raiden/pull/3916/commits/5792f97446809d652c9f2c1d2425a59070ae6951